lib/widgets/compose_box.dart

```diff
@@ -1509,7 +1509,7 @@ class _StreamComposeBoxBody extends _ComposeBoxBody {
   @override
   final StreamComposeBoxController controller;
 
-  @override Widget buildTopicInput() => _TopicInput(
+  @override Widget? buildTopicInput(BuildContext context) => _TopicInput(
```

Comment:
The change of the return type (from `Widget` to `Widget?`) is unrelated to the rest of this commit:

```
compose_box: Add BuildContext context arg to buildTopicInput
```